### ISSUE TITLE
Repository is poorly organized

### ISSUE THREAD

**Issue by TimidRobot** [creativecommons/cc-license-chooser#5]
The repository is poorly organized. It should use directories to better organize the files.

**Comment by svvarik** [comment]
Hi! This would be my first time contributing. Would I be able to work on this issue?

**Comment by kgodey** [comment]
Go ahead.

**Comment by tim-kilian** [comment]
The first thing could be to put the translations and the templates in separate direcories, so they are not at the top level anymore.

**Comment by ayushshivani** [comment]
@tim-kilian @kgodey If the issue is still up, I would like to work on it.

**Comment by tim-kilian** [comment]
Feel free.

I couldn't get these python scripts to run, thats why I didnt know if i would destroy some behavior while moving something.

**Comment by ayushshivani** [comment]
@tim-kilian I am having the same problem. How do I test my changes?

**Comment by kgodey** [comment]
@tim-kilian @ayushshivani what errors are you seeing?

**Comment by ayushshivani** [comment]
@kgodey I have made the changes. But am not able to test it? Any idea how to run the code in this repository or where it is used.

**Comment by kgodey** [comment]
@aayushdutt what have you tried so far to run it?

**Comment by tim-kilian** [comment]
I am stuck on this one while trying to run `make -s`:
```
from xml import xpath
ImportError: cannot import name xpath
```

**Comment by kgodey** [comment]
I got the Python scripts working on the branch associated with this pull request: https://github.com/creativecommons/cc-license-chooser/pull/10. I don't want to merge that yet (see PR for details) but you should be able to use the changes there to get set up locally.

**Comment by ahmadbilaldev** [comment]
Hi, 👋 I would like to work on this issue. Should I try testing it?

**Comment by kgodey** [comment]
@ahmadbilaldev go ahead, if you verify the output of https://github.com/creativecommons/cc-license-chooser/pull/10, that would be great.

**Comment by akmadian** [comment]
So, my understanding of this issue is that we should organize the repo to make the file structure cleaner, then run `update_jurisdictions.py`, and ensure that no behavior is broken because of the moved files. 

Are there any other scripts that need running? Any other things that need to be done before a PR is submitted that I've missed? I'm currently working through some pesky import errors, but I think I am making good headway.

**Comment by kgodey** [comment]
the code in https://github.com/creativecommons/cc-license-chooser/pull/10 fixes the import errors AFAIK.

**Comment by akmadian** [comment]
Yep, I looked through the fixed script, but I'm still trying to resolve some errors with BeautifulSoup and convert. Speaking of the fixes, would you want to see the python fixes in a PR, or should I keep the python scripts the same as what is currently in master for the PR?

Was my understanding of the task correct?

**Comment by kgodey** [comment]
Yes, please use 2.7; upgrading it should be a separate task. Your understanding of the task seems correct to me, tagging @TimidRobot in case they have something to add.

**Comment by akmadian** [comment]
I've moved templates and translations to their own dirs, and updated jurisdictions. Would you (maintainers) prefer that I take it further and start moving some of the js files into their own dir? If not, I can get a PR going pending your response!

**Comment by TimidRobot** [comment]
@akmadian I prefer smaller more frequent changes vs larger slower changes
